system/security/shared/SecureUser.hpp

```diff
@@ -50,6 +50,7 @@ class CSecureUser : implements ISecUser, implements ISecCredentials, public CInt
     CSecureUser(const char *name, const char *pw) : 
         m_name(name), m_pw(pw), m_authenticateStatus(AS_UNKNOWN), m_userID(0), m_status(SecUserStatus_Unknown), m_sessionToken(0)
     {
+        m_parameters.setown(createProperties(false));
```

Comment:
are we allowed to do this in the init list?

Reply:
Yes. Will make the change in all three constructors.